app/Http/Middleware/CheckAdmin.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\Auth;
+
+class CheckAdmin
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @param \Closure $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if (Auth::check() && Auth::user()->role_id === config('role.admin')) {
+            return $next($request);
+        } else return redirect()->route('home');
+    }
```

Comment:
```suggestion
        if (Auth::check() && Auth::user()->role_id === config('role.admin')) {
            return $next($request);
        } else {
            return redirect()->route('home');
        }
    }
```
sửa những chô tương tự